prow/cmd/splice/main.go

```diff
@@ -322,23 +324,26 @@ func main() {
 		if len(queue) == 0 {
 			continue
 		}
+		log.Infof("PRs in queue: %v", queue)
```

Comment:
Logging info at the start of the sync loop is not unreasonable, we are doing that in most core k8s controllers. We could also use more debugging logs across the board.